target_gsheet.py

```diff
@@ -48,18 +46,15 @@ def get_credentials():
     Returns:
         Credentials, the obtained credential.
     """
-    home_dir = os.path.expanduser('~')
-    credential_dir = os.path.join(home_dir, '.credentials')
-    if not os.path.exists(credential_dir):
-        os.makedirs(credential_dir)
-    credential_path = os.path.join(credential_dir,
-                                   'sheets.googleapis.com-singer-target.json')
+    client_secret = config.get('clientSecret', 'client_secret.json')
```

Comment:
should use `config.get('clientSecret', CLIENT_SECRET_FILE)` to avoid magic val

Reply:
Fixed